drivers/oled/glcdfont.c

```diff
@@ -3,6 +3,7 @@
 // Helidox 8x6 font with QMK Firmware Logo
 // Online editor: http://teripom.x0.com/
 
+#if OLED_FONT_SIZE == 1
```

Comment:
This I'm not a huge fan of.  

It might be worth renaming to `font_6x8` or what not.

Reply:
I don't think I quite get what you mean.
Do you mean removing the hard coded 1 and having something like `#if OLED_FONT_SIZE == OLED_FONT_6x8` with `OLED_FONT_6x8` being defined somewhere in the header?
Or do you mean removing the preprocessor directive altogether, name all of the fonts differently and have the compiler remove the unused fonts?